frontend/src/metabase/query_builder/components/EditableText/EditableText.tsx

```diff
@@ -1,25 +1,42 @@
-import React, { useState } from "react";
+import React, { useState, useRef } from "react";
 
-import { EditableTextRoot, EditableTextArea } from "./EditableText.styled";
+import {
+  EditableTextRoot,
+  EditableTextArea,
+  SharedStyles,
+  TEXT,
+} from "./EditableText.styled";
 
-import { KEY_ESCAPE } from "metabase/lib/keyboard";
-
-type TEXT = string | null | undefined;
```

Comment:
Could we get rid of this type entirely? `(value?: string | null)` would work if we need `null`